src/figuresClasses.ts

```diff
@@ -1,19 +1,94 @@
 export interface Figure {
+  shape:string,
+  color:string,
+  getArea: () => number,
+}
 
+function roundDownToHundredths(area:number):number {
+  return Math.floor(area * 100) / 100;
 }
 
-export class Triangle {
+export class Triangle implements Figure {
+  public shape = 'triangle';
 
-}
+  constructor(
+    public color:string,
+    public a:number,
+    public b:number,
+    public c:number,
+  ) {
+    if (!this.AreValidSides()) {
```

Comment:
should not be capitalized
```suggestion
    if (!this.areValidSides()) {
```